### ISSUE TITLE
Error building Maya plugin.

### ISSUE THREAD

**Issue by natwales** [PixarAnimationStudios/USD#711]
### Description of Issue
When running `build_usd.py` with the `--maya` flag I am getting the following errors. 

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
MAYA_Foundation_LIBRARY
    linked by target "usdMaya" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/usdMaya
MAYA_OpenMayaAnim_LIBRARY
    linked by target "usdMaya" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/usdMaya
MAYA_OpenMayaFX_LIBRARY
    linked by target "usdMaya" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/usdMaya
MAYA_OpenMayaRender_LIBRARY
    linked by target "px_vp20" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/px_vp20
    linked by target "usdMaya" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/usdMaya
MAYA_OpenMayaUI_LIBRARY
    linked by target "px_vp20" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/px_vp20
    linked by target "usdMaya" in directory /Users/user.name/Documents/learning/usd/USD/third_party/maya/lib/usdMaya
```
MAYA_OpenMaya_LIBRARY
    linked by target "px_vp20" in directory /Users/nat.wales/Documents/learning/usd/USD/third_party/maya/lib/px_vp20
    linked by target "usdMaya" in directory /Users/nat.wales/Documents/learning/usd/USD/third_party/maya/lib/usdMaya

### Steps to Reproduce
1. checkout build v18.11
2. add environment variables as described [here](https://graphics.pixar.com/usd/docs/Maya-USD-Plugins.html)
3. open python 2.7 terminal from anaconda, and confirm environment variables with `printenv`
4. run `python USD/build_scripts/build_usd.py /usr/local/USD --maya --maya-location /Applications/Autodesk/maya2018/`

### System Information (OS, Hardware)
Mac OS 10.13.6
### Package Versions
v18.11

**Comment by drwave** [comment]
The FindMaya.cmake module needs to be updated to find 2018 Maya.

I fixed it here, I will try and make a pull request this week.

In the meantime, if you look at cmake/modules/FindMaya.cmake it should be reasonably obvious what to change.

**Comment by jtran56** [comment]
Filed as internal issue #USD-4951.

**Comment by devGitHub48** [comment]
https://github.com/PixarAnimationStudios/USD/pull/570

**Comment by usmanredsoft** [comment]
Kindly upload Pixar USD maya Xcode project source?

**Comment by sunyab** [comment]
This was addressed with 2140815f in the dev branch. Marking this as pending release.

**Comment by sunyab** [comment]
This fix was released in v19.05, closing.
